docs/man/man1/kubectl-drain.1

```diff
@@ -18,9 +18,11 @@ Drain node in preparation for maintenance.
 .PP
 The given node will be marked unschedulable to prevent new pods from arriving.
 Then drain deletes all pods except mirror pods (which cannot be deleted through
-the API server).  If there are any pods that are neither mirror pods nor
-managed by a ReplicationController or DaemonSet, then drain will not delete any
-pods unless you use \-\-force.
+the API server) and DaemonSet\-managed pods.  If there are DaemonSet\-managed
+pods, drain will not proceed without \-\-ignore\-daemonsets, since those pods will
+be immediately replaced by the NodeController.  If there are any pods that are
```

Comment:
we never actually moved this to the NodeController; the daemons would be replaced by the DaemonSet controller.